simulator/scheduler/extender/resultstore/resultstore.go

```diff
@@ -6,15 +6,14 @@ import (
 
 	"golang.org/x/xerrors"
 	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog/v2"
 	extenderv1 "k8s.io/kube-scheduler/extender/v1"
 
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/scheduler/extender/annotation"
 )
 
 type Store interface {
-	AddStoredResultToPod(pod *v1.Pod)
+	AddStoredResultToPod(pod *v1.Pod) map[string]string
```

Comment:
```suggestion
	GetStoredResult(ns, name string) map[string]string
```
How about renaming this method since its usage has changed with this modification?
Also the comments on each method definition.